Aliases/az

```diff
@@ -0,0 +1 @@
+../Formula/azure-cli.rb
```

Comment:
These are supposed to be symlinks:
> Error: Non-symlink alias: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Aliases/az
Error: Non-symlink alias: /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Aliases/azure-cli@2.0